### ISSUE TITLE
Theme plugin: Enable accessibility checks and fix remaining a11y issues

### ISSUE THREAD

**Issue by christoph-jerolimov** [redhat-developer/rhdh-plugins#3591]
The theme plugin has playwright tests that include a bunch of accessibility assertions.

The tests runs `runAccessibilityTests` from `@workspaces/theme/packages/app-legacy/e2e-tests/utils/acessibility.ts` and `@workspaces/theme/packages/app/e2e-tests/utils/acessibility.ts`, but failures are ignored at the moment.

1. Remove the `skipViolationsAssert` attribute from the `options` parameter of `runAccessibilityTests`.
2. Run the expect statement always to check the validations.
3. Run the playwright tests via `CI=1 yarn test:e2e` and list the left over issues.
4. Try to resolve these issues in the code base. When unsure, or not all issues can be removed, create a PR anyway.
